pages/code_of_conduct.md

```diff
@@ -42,6 +42,7 @@ As someone who is part of this community, you agree that:
 - We respect people’s boundaries and identities.
 - We refrain from using language that can be considered offensive or oppressive (systemically or otherwise), eg. sexist, racist, homophobic, transphobic, ableist, classist, etc. - this includes (but is not limited to) various slurs.
```

Comment:
```suggestion
- We refrain from using language that can be considered offensive or oppressive (systemically or otherwise), eg. sexist, racist, homophobic, transphobic, ableist, classist, missionizing, etc. - this includes (but is not limited to) various slurs.
```